workers/tests/integrationtests/test_api.py

```diff
@@ -21,13 +21,18 @@ def setUp(self):
         self.result_schema = get_result_schema()
 
     def test_get_available_regions(self):
-        url = "/regions"
+        url = "/regions?geometry=true"
```

Comment:
I would name the parameter something like `asGeoJSON`.
Or `format=geojson` and default to `format=text`.